sdk/core/azure-core/src/tracing/tracing.cpp

```diff
@@ -21,41 +154,61 @@ namespace Azure { namespace Core { namespace Tracing { namespace _internal {
   const TracingAttributes TracingAttributes::RequestId("requestId");
   const TracingAttributes TracingAttributes::HttpStatusCode("http.status_code");
 
+  std::string TracingContextFactory::BuildUserAgent(
```

Comment:
Please create a work item to combine this logic, and the logic we have for TelemetryPolicy, so there is minimum code repetition.
We are updating core anyways, either we re-ship now, or next month, so please have this code in one place, and make both TelemetryPolicy and this place to call the same function.

Reply:
Fair, but functionally TelemetryPolicy is dead code waiting to be removed.

Reply:
Until we ship core 2.0, it will have to be there, we are forever backwards compatible within major versions.